src/foam/u2/wizard/wizardlet/SuccessWizardletView.js

```diff
@@ -53,6 +54,12 @@ foam.CLASS({
         .start()
           .addClass('h200')
           .add(this.message$)
+        .end()
+        .start()
+          .callIf(this.confirmationNumber, function() {
```

Comment:
`this.confirmationNumber`
OR
`this.confirmationNumber$`
## what's the difference?